src/pages/Categories.js

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { statusCheck } from '../redux/categories/categories';
 
-const Categories = () => <button type="button"> Check status</button>;
+const Categories = () => {
+  const status = useSelector((state) => state.Categories);
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(statusCheck());
+        }}
+      >
+        Check status
```

Comment:
- [ ] Make sure the button "Check status" works as expected: "Under Construction"
![Capture](https://user-images.githubusercontent.com/63972670/208889569-76737424-7bc7-4f87-9424-55bbb0852d57.PNG)

Reply:
Thankyou ✔